src/compiler/core.ts

```diff
@@ -1730,13 +1730,18 @@ namespace ts {
 
     /* @internal */
     export function startsWith(str: string, prefix: string): boolean {
-        return str.lastIndexOf(prefix, 0) === 0;
+        return str.indexOf(prefix) === 0;

```

Comment:
This will change the performance -- previously it would only look at the start, now it will look everywhere and then check whether the result was the start.